### ISSUE TITLE
ldes-server start fail 

### ISSUE THREAD

**Issue by zargha** [Informatievlaanderen/VSDS-LDESServer4J#187]
### The Problem
While testing out the e2e gipod test e2e-test/use-cases/gtfs-and-rt/1.ingest-ldes, the `LDES server` fails to run.

### Environment
**Components under test**
- ldes server
- mongodb
- gtfs2ldes-js
- nifi-workflow
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.4 LTS
Release:        20
```
```
$ docker --version
Docker version 20.10.17, build 100c701
$ docker compose version
Docker Compose version v2.6.0
```
```
Tested with Docker image tag: 20220816T1248 and 20220823T1603
```
### Error logs
```
2022-08-26 10:05:46.582  WARN 1 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'geospatialBucketiser' defined in URL [jar:file:/lib/ldes-fragmentisers-geospatial-jar-with-dependencies.jar!/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/geospatial/bucketising/GeospatialBucketiser.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.boot.context.properties.ConfigurationPropertiesBindException: Error creating bean with name 'geospatialConfig': Could not bind properties to 'GeospatialConfig' : prefix=geospatial, ignoreInvalidFields=false, ignoreUnknownFields=true; nested exception is org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'geospatial.max-zoom-level' to int
2022-08-26 10:05:46.587  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2022-08-26 10:05:46.599  INFO 1 --- [           main] ConditionEvaluationReportLoggingListener : 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2022-08-26 10:05:46.609 ERROR 1 --- [           main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to bind properties under 'geospatial.max-zoom-level' to int:

    Property: geospatial.max-zoom-level
    Value: ""
    Origin: System Environment Property "GEOSPATIAL_MAXZOOMLEVEL"
    Reason: failed to convert java.lang.String to int (caused by java.lang.IllegalArgumentException: A null value cannot be assigned to a primitive type)

Action:

Update your application's configuration
```

**Comment by rorlic** [comment]
Indeed, this is currently an issue. The workaround is simple, add the following to your `env.user` file:
```
GEOSPATIAL_MAXZOOMLEVEL=0 # Workaround (Failed to bind properties under 'geospatial.max-zoom-level' to int)
```

**Comment by zargha** [comment]
Update env.user with GEOSPATIAL_MAXZOOMLEVEL=0 
Getting this error now:
```
alifies as autowire candidate. Dependency annotations: {}
2022-08-26 10:11:46.769  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2022-08-26 10:11:46.807  INFO 1 --- [           main] ConditionEvaluationReportLoggingListener : 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2022-08-26 10:11:46.838 ERROR 1 --- [           main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 1 of constructor in be.vlaanderen.informatievlaanderen.ldes.server.fragmentisers.geospatial.bucketising.GeospatialBucketiser required a bean of type 'be.vlaanderen.informatievlaanderen.ldes.server.fragmentisers.geospatial.bucketising.CoordinateConverter' that could not be found.


Action:

Consider defining a bean of type 'be.vlaanderen.informatievlaanderen.ldes.server.fragmentisers.geospatial.bucketising.CoordinateConverter' in your configuration.
```

**Comment by Yalz** [comment]
Hi Zargham, could you post your full environment properties here?

**Comment by zargha** [comment]
> Hi Zargham, could you post your full environment properties here?

# GTFS2LDES compose env vars
GTFS2LDES_TAG=20220714t1136
RUN_ON_LAUNCH=true
THROTTLE_RATE=100
GTFS2LDES_DATA_FOLDER=./data/gtfs/nmbs/lc/ 
GTFS_SOURCE=https://sncb-opendata.hafas.de/gtfs/static/c21ac6758dd25af84cca5b707f3cb3de
GTFS_CRON=0 0 3 * * *
GTFSRT_SOURCE=https://sncb-opendata.hafas.de/gtfs/realtime/c21ac6758dd25af84cca5b707f3cb3de
GTFSRT_CRON=*/30 * * * * *
AUTH_HEADER=
AUTH_HEADER_VALUE=

# Workflow compose env vars
PAT_READ_PACKAGES=
LDES_CLIENT_NAR_TAG=20220818.141937-26
NIFI_UI_PORT=8443
SINGLE_USER_CREDENTIALS_USERNAME=
SINGLE_USER_CREDENTIALS_PASSWORD=
NIFI_WORKFLOW_LISTEN_PORT=9005
NIFI_JVM_HEAP_INIT=8g
NIFI_JVM_HEAP_MAX=8g

# Server compose env vars
SPRING_DATA_MONGODB_DATABASE=test
LDES_SERVER_TAG=20220823T1603 # 20220816T1248 
LDES_SERVER_PORT=8080
LDES_COLLECTIONNAME="connections"
LDES_MEMBERTYPE="http://semweb.mmlab.be/ns/linkedconnections#Connection"
# LDES_SHAPE is not used so left blank intentionally:
LDES_SHAPE=
VIEW_TIMESTAMPPATH="http://www.w3.org/ns/prov#generatedAtTime"
VIEW_VERSIONOFPATH=http://purl.org/dc/terms/isVersionOf
TIMEBASED_MEMBERLIMIT=100
GEOSPATIAL_MAXZOOMLEVEL=0

# Mongo compose env vars
MONGODB_TAG=5.0.9
MONGODB_PORT=27017
MONGODB_DATA_FOLDER=./data/gtfs/nmbs/mongo.db/

**Comment by rorlic** [comment]
Again, there is a workaround: for now, you need go add the GEOSPATIAL_BUCKETISER=lambert72 config setting. @Yalz can you please double check and confirm the above?

**Comment by WLefever-Cegeka** [comment]
I fixed the issue in #192 ? Can you try again with the current tag: 20220831t1755?

**Comment by zargha** [comment]
The issue has been fixed in Docker image tag `20220923t0958`
